components/pedigree/resources/src/main/resources/pedigree/view/person.js

```diff
@@ -998,7 +1008,12 @@ define([
             return this._getGeneArray("rejected");
         },
 
-        // returns null if geneID is not presemnt, or gene status if it is
+        // used by controller in conjuntion with setCandidateGenes
```

Comment:
we have that comment for `getCausalGenes()` and `getRejectedGenes()`, if it is valid there why it is not valid for `getCarrierGenes()`?